### DISCUSSION TITLE
Default DAB to be disabled by default in pufferfish.yml

### DISCUSSION THREAD

**Discussion by ghost** [PurpurMC/Purpur#941] [Feature Requests]
As said in the title, i think the default Pufferfish configs for Purpur should have DAB disabled. DAB has been known to cause certain farms (zombified piglin, raid farms) to not work as intended. As the general audience for Purpur differs a lot from the audience Pufferfish's default configs are intended for, a lot of the people that come looking for features then come to realize their farms aren't working the same as on Paper will be under the impression that Purpur broke these farms, and likely won't do research before switching back to Paper. The support Discord for Purpur also receives quite a few questions that are resolved by disabling DAB. 

Tweaking these values to make the farms in question work will likely make DABs benefit negligible, and the people that need DAB are likely knowledgable enough to toggle it on themselves.

**Comment by BillyGalbreath** [comment]
We don't change anything from upstream by default. This has always been one of our main goals with this project in order to not surprise anyone with unexpected changes. Just turn off DAB if you don't want DAB. Simple.
